A/Atomsk/build_tarballs.jl

```diff
@@ -3,29 +3,43 @@
 using BinaryBuilder, Pkg
 
 name = "Atomsk"
-version = v"0.11.2"
+version = v"0.12.0"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/pierrehirel/atomsk.git", "3333858281e0ebd6279825b83ab871cf4d050a8d")
+    GitSource("https://github.com/pierrehirel/atomsk.git", "84f60a20c5b814fec03bffe1bccc3daaed0fc65d"),
+    DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/atomsk/src/
-mkdir -p ${bindir}
+cd $WORKSPACE/srcdir/atomsk
+if [[ "$nbits" == 32 || "$target" == *apple* ]]; then
+    atomic_patch -p1 ../patches/atomsk_32.patch
+else
+    atomic_patch -p1 ../patches/atomsk_64.patch
+fi
+if [[ "$target" == *mingw* ]]; then
+    ln -s $prefix/bin/libatomsk.dll $prefix/lib/libatomsk.dll
+fi
+cd src
+
 # The makefile doesn't handle parallel builds
-make atomsk BIN="atomsk${exeext}"
-make install INSTPATH=${prefix} BIN="atomsk${exeext}"
+if [[ ! -d "$bindir" ]]; then
+    mkdir ${bindir}
+fi
+make atomsk
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms(; experimental=true)
-platforms = expand_gfortran_versions(platforms)
+platforms = supported_platforms()
+platforms = expand_cxxstring_abis(platforms)
```

Comment:
Why?

Reply:
I just saw it in the LAMMPS build_tarballs.jl. Is one preferred over the other?

Reply:
It isn't about preference, it's about one thing is needed or not: https://docs.binarybuilder.org/stable/building/#Expanding-C-string-ABIs-or-libgfortran-versions.  Here you have to expand the libgfortran version because we're linking to libgfortran, this code seems to be written entirely in Fortran so expanding the C++ string ABI makes no sense.

Reply:
Ah, I see. That makes sense.